src/components/carousel/carousel.spec.tsx

```diff
@@ -26,6 +29,27 @@ function renderCarousel(props: CarouselProps, renderer = mount) {
 }
 
 describe("Carousel", () => {
+  describe("Deprecation warning", () => {
+    it("when user uses the component, a deprecation warning is raised in the console", () => {
```

Comment:
suggestion: add `only once` to the test title

The test seems to also check if we only log out the deprecation warning once, despite the component being used twice, so I think it would be good to add that aspect to the title

```suggestion
    it("when user uses the component, a deprecation warning is raised only once in the console", () => {
```